.travis.yml

```diff
@@ -1,5 +1,5 @@
 os: linux
-sudo: required
+# sudo: required # deprecated as of 2021
```

Comment:
Hey Sebastian, made some mistake here with your comment, can it be more descriptive? @Boccca2014